app/views/components/_navigation-sidebar.html.erb

```diff
@@ -0,0 +1,48 @@
+<%
+  related_items ||= []
+  topics ||= []
+  publishers ||= []
+  collections ||= []
+  policies ||= []
+  related_content = [
+    {"related_items" => related_items},
+    {"topics" => topics},
+    {"publishers" => publishers},
+    {"collections" => collections},
+    {"policies" => policies}
+  ]
+%>
+<% if related_content.map(&:values).flatten.any? %>
+  <aside class="app-c-navigation-sidebar" role="complementary">
+    <% related_content.each do |section| %>
+      <% section.select do |section_title, links| %>
+
+        <% if section_title === "related_items" %>
+          <h2 id="nav-sidebar-<%= section_title %>" class="app-c-navigation-sidebar__main-heading" ><%= t('components.navigation_sidebar.related_content') %></h2>
+          <nav role="navigation" class="app-c-navigation-sidebar__nav-section" aria-labelledby="nav-sidebar-<%= section_title %>" %>
+            <ul>
+              <% links.each do |link| %>
+              <li class="app-c-navigation-sidebar__link">
+                <%= link_to link[:text], link[:href], class: "app-c-navigation-sidebar__related-link" %>
+              </li>
+              <% end %>
+            </ul>
+          </nav>
+
+        <% elsif links.any? %>
+          <h3 id="nav-sidebar-<%= section_title %>" class="app-c-navigation-sidebar__sub-heading"><%= t('components.navigation_sidebar.' + section_title) %></h3>
```

Comment:
Should we give the different subheading/link blocks classes related to what they are instead of generic ones (__topic, __from etc)? I'm unsure - I feel compelled to do it but since they all look the same perhaps it's unnecessary clutter.

Reply:
I did think about this, but ended up giving the same classes seeing as they're all styled the same (apart from the related content section at the top).